### PR TITLE
Issue #96 Authentication fields for website/services

### PR DESCRIPTION
**Pull request by Laupe**
- File functions.inc.php - modified and added features psm_curl_get parameters for website_username and website_password
- File functions.inc.php - new features and psm_password_encrypt psm_password_decrypt to encrypt and decrypt passwords
- Adjusted basic language EN and supplemented by new language CZ phrase
- ConfigController.php - adjusted executeIndex method of password encryption key for each installation separately + executeSave method of saving the settings + added labels to methods getLabels
- AbstractServerController.php - adjusted method getServers a login password of the name
- ServerController.php - adjusted executeEdit method for storing information about name and password + executeSave method of storage in a database + method getLabels on labels
- Installer.php - adjusted method install a new configuration values ​​for the encryption key passwords + method installTables a new table structure servers + method upgrade320 the new structure
- StatusUpdater.php - adjusted method update the transmission website_username and website_password + method updateWebsite adjusted parameters passed to the function psm_curl_get
- Modified template: config/config.tpl.html, server/update.tpl.html, view.tpl.html
- Adapted javascript scripts.js

### CODE REVIEW

File: src/psm/Util/Install/Installer.php [hunk 1]
```diff
@@ -146,6 +146,7 @@ public function install() {
 					('sms_from', '1234567890'),
 					('pushover_status', '0'),
 					('pushover_api_token', ''),
+					('password_encrypt_key', " . sha1(microtime()) . "),
```
Comment (sadortun) [inline src/psm/Util/Install/Installer.php]: fixed, misisng `''`

Comment (Laupe) [inline src/psm/Util/Install/Installer.php]: ohh sorry I fix it now! Thanks

Comment (Laupe) [inline src/psm/Util/Install/Installer.php]: @sadortun fixed ;)

**Comment by sadortun** [comment]
Thanks!

I've added some obfuscation to avoid display the plain text password in the form.

**Pull request merged**